### ISSUE TITLE
Planet's Frontend

### ISSUE THREAD

**Issue by ashmaroli** [planetjekyll/planetjekyll.github.io#1]
Hi Gerald,
First of all, well done on maintaining a database-of-sorts on various accessories related to Jekyll (and ruby..) :+1:  :+1: 

I have a few issues mainly with how the site 'looks and feels' to 'me'.

- The *Planet's* Home Page looks rather unstyled or very minimally styled.
- The contents of `.cards` be it the divs on the homepages / themes / plugins are aligned to the left, with loads of empty space to the right side. A nice alternative would be to have the parent-container fixed at a certain width (yet responsive and fluid) and center-aligned.
- The individual 'cards' would look better with a fixed height, some subtle texture and optionally a drop-shadow (yes, Material Design principle).

Some additional eye-candy:
- The cards could flip along the Y-axis (CSS Transitions) onMouseover and show more details regarding the theme/plugin.
- Consider using one of the theme's showcased in your site to style the entire *Planet*.
- Specifically to theme-gems, you could have a 'corner-ribbon' that says "Github Pages" or similar to denote those are maintained by GitHub and can be used on GitHub (latest release of `github-pages` gem.).
- Specifically to themes or plugins, a corner-ribbon to denote "Popular" (based on stars) or "Official" (based on author/owner).

Easier said than done, I agree. :smiley:

**Comment by geraldb** [comment]
@ashmaroli  I agree. It is in no way perfect. There is only so much I can do. The idea of course is always to make the site and pages better. Thanks for your ideas and details comments on how to improve the pages. I am offline for a couple of weeks so this has to wait for next year (unless somebody helps out). Thanks for your kind words. Have a great 2017! Cheers.

**Comment by ashmaroli** [comment]
Season Greetings to you too :smiley: I'll chip in a few PRs as when I get time too.. will earmark it TODO in 2017 for now..

**Comment by geraldb** [comment]
@ashmaroli  Again you are more than welcome to chip in. No rush. Happy new year. Prosit 2017! PS: I am back (online) in February 2017. Cheers.
